engine/schema/resources/META-INF/db/schema-41110to41120.sql

```diff
@@ -27,4 +27,28 @@ INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid,hypervisor_type, hypervis
 ALTER TABLE `cloud`.`user_vm_clone_setting`
 ADD COLUMN `id` BIGINT(20) UNSIGNED NOT NULL AUTO_INCREMENT FIRST,
 DROP PRIMARY KEY,
-ADD PRIMARY KEY (`id`);
\ No newline at end of file
+ADD PRIMARY KEY (`id`);
+
+CREATE TABLE `cloud`.`direct_download_certificate` (
+  `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
+  `uuid` varchar(40) NOT NULL,
+  `alias` varchar(255) NOT NULL,
+  `certificate` text NOT NULL,
+  `hypervisor_type` varchar(45) NOT NULL,
+  `zone_id` bigint(20) unsigned NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `i_direct_download_certificate_alias` (`alias`),
+  KEY `fk_direct_download_certificate__zone_id` (`zone_id`),
+  CONSTRAINT `fk_direct_download_certificate__zone_id` FOREIGN KEY (`zone_id`) REFERENCES `data_center` (`id`) ON DELETE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `cloud`.`direct_download_certificate_host_map` (
+  `id` bigint(20) unsigned NOT NULL AUTO_INCREMENT,
+  `certificate_id` bigint(20) unsigned NOT NULL,
+  `host_id` bigint(20) unsigned NOT NULL,
+  PRIMARY KEY (`id`),
+  KEY `fk_direct_download_certificate_host_map__host_id` (`host_id`),
+  KEY `fk_direct_download_certificate_host_map__certificate_id` (`certificate_id`),
+  CONSTRAINT `fk_direct_download_certificate_host_map__host_id` FOREIGN KEY (`host_id`) REFERENCES `host` (`id`) ON DELETE CASCADE,
+  CONSTRAINT `fk_direct_download_certificate_host_map__certificate_id` FOREIGN KEY (`certificate_id`) REFERENCES `direct_download_certificate` (`id`) ON DELETE CASCADE
```

Comment:
@nvazquez move this to a new 4.11.2.0-4.11.3.0 db upgrade path (for that please submit a separate pr)